roles/openshift_setup/tasks/main.yml

```diff
@@ -183,8 +183,14 @@
       spec:
         repositoryDigestMirrors: "{{ cifmw_openshift_setup_digest_mirrors }}"
 
-- name: Patch network operator when using OVNKubernetes backend
-  ansible.builtin.import_tasks: patch_network_operator.yml
+- name: OpenShift tweaks
+  when: not cifmw_openshift_setup_dry_run
```

Comment:
with a "when", please move to `include_tasks` to improve speed:
https://ci-framework.readthedocs.io/en/latest/development/01_guidelines.html#include-vs-import (check the "rule of thumb")

Reply:
(note to self: I have to update the doc to effectively cover the `when` usage + include_*)